src/app/shared/http-services/group-permissions.service.ts

```diff
@@ -5,22 +5,35 @@ import { map } from 'rxjs/operators';
 import { appConfig } from '../helpers/config';
 import { assertSuccess, SimpleActionResponse } from './action-response';
 import * as D from 'io-ts/Decoder';
-import { pipe } from 'fp-ts/function';
-import { decodeSnakeCase } from 'src/app/shared/operators/decode';
-import { permissionsDecoder, Permissions } from '../helpers/group-permissions';
+import { permissionsDecoder } from 'src/app/modules/item/helpers/item-permissions';
+import { pipe } from 'fp-ts/lib/function';
```

Comment:
I know the autocomplete generates an import this way, but we should import from `fp-ts/function` to let the bundler decide whether to use the esm or cjs pkg build

Reply:
fixed